py_zipkin/zipkin.py

```diff
@@ -4,10 +4,9 @@
 import time
 from collections import namedtuple
 
+from py_zipkin import thread_local
```

Comment:
Seems a little weird to import `thread_local` here. What if you were to just create a new (sub?)class of `ThreadLocalStack` that is specific to the span storage? We could also have a subclass specific to zipkin attrs, which I think would simplify the defaulting logic as well